src/components/Player/Pages/Histograms/Histograms.jsx

```diff
@@ -1,13 +1,14 @@
-import React from 'react';
 import PropTypes from 'prop-types';
+import React from 'react';
```

Comment:
FYI: Just my formatting addon sorting imports. if I should change it back let me know.